scripts/upgrade

```diff
@@ -83,10 +86,28 @@ ynh_secure_remove ${tmpdir}
 # RELOAD RELEVANT SERVICES
 #=================================================
 
-ynh_systemctl reload php5-fpm
-ynh_systemctl reload nginx
+systemctl reload php5-fpm
+systemctl reload nginx
+
+###  Make sure that the yunohost services have a description and need-lock enabled
+
+# main service
+
+yunohost service remove $service_name
+yunohost service add $service_name --description "manages the VPN client" --need_lock
+yunohost service start $service_name
```

Comment:
hmmm, i dunno if we need to "start" the service here ? Shouldn't it be already started ? Will it accept to "start" if it's already started (compared to restart ?) Also in fact we seem to be restarting it later ?